include/bpf_helpers.h

```diff
@@ -41,6 +41,7 @@
 
 #define bpf_map_def _ebpf_map_definition_in_file
 #include "ebpf_nethooks.h"
+#include "ebpf_xdp_test_hooks.h"
```

Comment:
I think test headers shouldn't be included by non-test headers.
Hence recommend removing this line and including it by any test files that need it.